14-microbiome-diversity.Rmd

```diff
@@ -106,15 +106,33 @@ between communities (samples).
 Technically, beta diversities are usually represented as `dist`
 objects, which contain triangular data describing the distance between
 each pair of samples. These distances can be further subjected to
-ordination with methods such as multi-dimensional scaling (also known
-as PCoA) to retrieve reduced dimensions for further evaluation or
-visualization. [TODO we could here instead link to ordination chapter?]
+ordination. Ordination is a common concept in ecology that aims to
+reduce the dimensionsionality of the data for further evaluation or
+visualization. Ordination techniques aim to capture as much of
+essential information in the data as possible in a lower dimensional
+representation.  Dimension reduction is bound to loose information but
+the common ordination techniques aim to preserve relevant information
+of sample similarities in an optimal way, which is defined in
+different way by different methods. [TODO add references and/or link
+to ordination chapter instead?]
+
+Some of the most common ordination methods in microbiome research
+include Principal Component Analysis (PCA), metric and non-metric
+multi-dimensional scaling (MDS, NMDS), The MDS methods is also known
+as Principal Coordinates Analysis (PCoA). Other recently popular
+techniques include t-SNE and UMAP. 
+
+Some of the above-mentioned ordination methods (PCA, MDS/PCoA) allow
+the estimation of explained variance in the data for each ordination
+axis; this is informative to show in the figures when it is defined
+and available.
+
 
 
 ### Estimating beta diversity
 
-In the following examples distances are calculated by variable
-functions supplied to the `FUN` argument. The function can defined by
+In the following examples dissimilarities are calculated by 
```

Comment:
diversity/dissimilarities/distances are sometimes stand-ins for each other.

Can you add an explanation here how it is supposed to be understand in the context of `MiaBook`?

Reply:
Ah, this is quite essential difference technically since "distance" is a mathematically well-defined concept and many ecological beta diversity measures, such as Bray-Curtis, are not proper distances. It is more accurate to call them dissimilarities, and it is in a strict sense wrong to call them distances. It is not just semantics and I think we should aim to use technically correct vocabulary that is accepted also by the comp/stat oriented members of our community.

Reply:
Thats a good explanation. Can you add this?

Reply:
Added, in the next update to the PR.

Reply:
Added.